qml/Main.qml

```diff
@@ -292,28 +286,28 @@ Window {
                 The panel stays in html5, too, for ease of integration.
                 **/
 
-                WebContext {
+                WebEngineProfile {
                     id: webcontext
-                    userAgent: navApp.appUA
+                    httpUserAgent: navApp.appUA
                     userScripts: [
-                        Oxide.UserScript {
-                            context: mainPageStack.usContext
-                            url: Qt.resolvedUrl("js/oxide.js")
-                        }
+                        // WebEngineScript {
```

Comment:
Remove commented code. If needed there is git revert :P

Reply:
You are right as always